### PR TITLE
Add archbius tools to backend

### PR DESCRIPTION
**Pull request by kyleaitken**
This PR provides an implementation for an Archibus proof of concept, allowing users to make workspace bookings to the archibus dev server

### CODE REVIEW

File: app/frontend/src/i18n/locales/translations.json [hunk 1]
```diff
@@ -458,5 +458,29 @@
...
         "en": "Skip tutorial",
         "fr": "Passer le tutoriel"
+    },
+    "archibus": {
+        "en": "Archibus",
+        "fr": "Archibus"
+    },
+    "booking.success": {
+        "en": "Your workspace has been reserved successfully for the date: ",
+        "fr": ""
```
Comment (KingBain) [inline app/frontend/src/i18n/locales/translations.json:468]:
```suggestion
"fr": "Votre espace de travail a été réservé avec succès pour la date :" 
```

File: app/frontend/src/i18n/locales/translations.json [hunk 2]
```diff
@@ -458,5 +458,29 @@
...
+    },
+    "booking.fail": {
+        "en": "There was an issue reserving your workspace: ",
+        "fr": ""
```
Comment (KingBain) [inline app/frontend/src/i18n/locales/translations.json:472]:
```suggestion
"fr": "Il y a eu un problème lors de la réservation de votre espace de travail : "
```

File: app/frontend/src/i18n/locales/translations.json [hunk 3]
```diff
@@ -458,5 +458,29 @@
...
+    },
+    "booking.complete": {
+        "en": "COMPLETE RESERVATION",
+        "fr": ""
```
Comment (KingBain) [inline app/frontend/src/i18n/locales/translations.json:476]:
```suggestion
"fr": "COMPLÉTER LA RÉSERVATION"
```

File: app/frontend/src/i18n/locales/translations.json [hunk 4]
```diff
@@ -458,5 +458,29 @@
...
+    },
+    "aria.expand.floorPlan": {
+        "en": "Expand floor plan image",
+        "fr": ""
```
Comment (KingBain) [inline app/frontend/src/i18n/locales/translations.json:480]:
```suggestion
"fr": "Agrandir l'image du plan d'étage"

```

File: app/frontend/src/i18n/locales/translations.json [hunk 5]
```diff
@@ -458,5 +458,29 @@
...
+    },
+    "archibus.floorPlan": {
+        "en": "Archibus floor plan",
+        "fr": ""
```
Comment (KingBain) [inline app/frontend/src/i18n/locales/translations.json:484]:
```suggestion
"fr": "Plan d'étage Archibus"

**Comment by KingBain** [comment]
Added french translations, using GPT(best effort) 

@kyleaitken  [if they're acceptable, github should make it painless to accept the suggestions. ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes)

**Comment by GuillaumeTurcotte** [comment]
This is closed, this was merged in the `dev-update` branch since I had other changes to consider (the tools rework).

**Pull request closed without merging**